docs/src/app/(public)/(content)/react/components/navigation-menu/demos/hero/css-modules/index.module.css

```diff
@@ -121,13 +121,9 @@
   transition-property: opacity, transform, width, height;
   transition-duration: var(--duration);
   transition-timing-function: var(--easing);
-  width: max-content;
+  width: var(--popup-width);
   height: var(--popup-height);
 
-  @media (min-width: 500px) {
-    width: var(--popup-width);
-  }
```

Comment:
This can be considered a breaking change. On <500px, the popup collapses in size when animating out